manifests/postgresql_settings/table_settings.pp

```diff
@@ -7,6 +7,8 @@
                                                                         {'default_value' => 0.80}),
   Optional[Float[0,1]] $reports_autovacuum_vacuum_scale_factor  = lookup('pe_databases::postgresql_settings::reports_autovacuum_vacuum_scale_factor',
                                                                         {'default_value' => 0.01}),
+  Optional[Float[0,1]] $fact_paths_autovacuum_vacuum_scale_factor = lookup('pe_databases::postgresql_settings::factsets_autovacuum_vacuum_scale_factor',
```

Comment:
No need for the lookup that is a compatibility layer for older parameters.  It should look like the params below here.